### PR TITLE
Normalize CI config

### PR DESCRIPTION
**Pull request by szepeviktor**
As it is a YAML file.

- every string gets double-quotes (even shell strings)
- introduce `true`

### CODE REVIEW

**Comment by szepeviktor** [comment]
Silly-billy Viktor is no developer!

Not it is fixed. With lock file fixed it will be 🍏

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,68 +1,71 @@
...
-            -   name: Install Composer dependencies
-                uses: ramsey/composer-install@v2
+            -   name: "Install Composer dependencies"
+                uses: "ramsey/composer-install@v2"
 
-            -   name: Install CoversValidator Composer dependencies
-                if: matrix.skip_static_checks != 'true'
-                uses: ramsey/composer-install@v2
+            -   name: "Install CoversValidator Composer dependencies"
+                if: "! matrix.skip_static_checks"
```
Comment (theofidry) [inline .github/workflows/ci.yaml:39]: I think you need to keep the `!= 'true'` there: we are not checking real booleans but against a `'true'` string

Comment (szepeviktor) [reply .github/workflows/ci.yaml:39]: That true string is gone.
We now have a real YAML true!

Comment (szepeviktor) [reply .github/workflows/ci.yaml:39]: ... and work as expected:

kép

steps are being skipped 🥳

Comment (szepeviktor) [reply .github/workflows/ci.yaml:39]: Got it.
The first "Mark ..." step needs no `!`

**Comment by szepeviktor** [comment]
All green! 🍏

**Comment by theofidry** [comment]
Thank you @szepeviktor!

**Comment by szepeviktor** [comment]
It feels very good to work for you.

**Pull request merged**